calcite-adapter/src/test/java/software/amazon/documentdb/jdbc/query/DocumentDbQueryMappingServiceBasicTest.java

```diff
@@ -1633,20 +1633,14 @@ void testQueryAndWithTypes() throws SQLException {
         Assertions.assertEquals(
                 BsonDocument.parse(
                         "{\"$project\": {"
-                                + "\"EXPR$0\": {\"$cond\": [{\"$and\": [{\"$eq\": [true, {\"$cond\": ["
-                                + "{\"$and\": [{\"$gt\": [\"$field\", null]}, "
-                                + "{\"$gt\": [{\"$date\": \"2021-01-01T00:00:00Z\"}, null]}]}, "
-                                + "{\"$gt\": [\"$field\", {\"$date\": \"2021-01-01T00:00:00Z\"}]}, null]}]}, "
-                                + "{\"$eq\": [true, {\"$cond\": [{\"$and\": [{\"$gt\": [\"$field\", null]}, "
-                                + "{\"$gt\": [{\"$date\": \"2020-02-01T00:00:00Z\"}, null]}]}, "
-                                + "{\"$lt\": [\"$field\", {\"$date\": \"2020-02-01T00:00:00Z\"}]}, null]}]}]}, true, "
-                                + "{\"$cond\": [{\"$or\": [{\"$eq\": [false, "
-                                + "{\"$cond\": [{\"$and\": [{\"$gt\": [\"$field\", null]}, "
-                                + "{\"$gt\": [{\"$date\": \"2021-01-01T00:00:00Z\"}, null]}]}, "
-                                + "{\"$gt\": [\"$field\", {\"$date\": \"2021-01-01T00:00:00Z\"}]}, null]}]}, "
-                                + "{\"$eq\": [false, {\"$cond\": [{\"$and\": [{\"$gt\": [\"$field\", null]}, "
-                                + "{\"$gt\": [{\"$date\": \"2020-02-01T00:00:00Z\"}, null]}]}, "
-                                + "{\"$lt\": [\"$field\", {\"$date\": \"2020-02-01T00:00:00Z\"}]}, null]}]}]}, false, null]}]}"
+                                + "\"EXPR$0\": {\"$cond\": [{\"$and\": [{\"$eq\": [true, null]}, "
```

Comment:
Concerned about this one and there isn't a server test similar enough to it yet. I don't see the date literals anymore in the new output. testQuerySelectLogicNullAndTypes and testQuerySelectLogicNulls in DocumentDbStatementBasicTest are similar but are not testing the same field against two different literals.